metrics-facade-base/src/main/java/com/ringcentral/platform/metrics/producers/AbstractMemoryMetricsProducer.java

```diff
@@ -0,0 +1,36 @@
+package com.ringcentral.platform.metrics.producers;
+
+import com.ringcentral.platform.metrics.MetricModBuilder;
+import com.ringcentral.platform.metrics.names.MetricName;
+
+import java.lang.management.MemoryMXBean;
+import java.lang.management.MemoryPoolMXBean;
+import java.util.List;
+
+import static java.util.Objects.requireNonNull;
+
+public abstract class AbstractMemoryMetricsProducer extends AbstractMetricsProducer implements MemoryMetricsProducer {
+    public static final MetricName DEFAULT_NAME_PREFIX = MetricName.of("Memory");
+
+    protected static final String INIT_MESSAGE = "The amount of memory in bytes that the Java virtual machine initially requests from the operating system for memory management";
```

Comment:
This name is inconsistent with the DESCRIPTION constants used for the same purpose.

Reply:
Fixed, thank you.